MapboxNavigation/NavigationMapView.swift

```diff
@@ -12,11 +12,11 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     
     // MARK: Class Constants
     
-    struct FrameIntervalOptions {
-        static let durationUntilNextManeuver: TimeInterval = 7
-        static let durationSincePreviousManeuver: TimeInterval = 3
-        static let defaultFramesPerSecond = MGLMapViewPreferredFramesPerSecond.maximum
-        static let pluggedInFramesPerSecond = MGLMapViewPreferredFramesPerSecond.lowPower
+    public struct FrameIntervalOptions {
+        public static let durationUntilNextManeuver: TimeInterval = 7
+        public static let durationSincePreviousManeuver: TimeInterval = 3
+        public static let defaultFramesPerSecond = MGLMapViewPreferredFramesPerSecond.maximum
```

Comment:
Of these enumeration values, only `defaultFramesPerSecond` is used in CarPlay code. Let’s extract `defaultFramesPerSecond` out of this struct into a standalone constant and keep this struct internal.